docs/themes/porter/layouts/index.html

```diff
@@ -200,12 +200,21 @@ <h4>Work on Porter with us! 💖</h4>
     </div>
   </div>
 
-  <footer class="row" id="footer">
-    <div class="small-12 medium-12 large-12 columns cncf">
-      <a href="https://www.cncf.io/" title="Cloud Native Computing Foundation" target="_blank">
-        <img src="/src/img/cncf-color.png" alt="We are a Cloud Native Computing Foundation sandbox project">
-        <p>We are a <strong>Cloud Native Computing Foundation</strong> sandbox project.</p>
-      </a>
+  <footer id="footer">
+    <div class="row">
+      <div class="small-12 medium-12 large-12 columns cncf">
+        <a href="https://www.cncf.io/" title="Cloud Native Computing Foundation" target="_blank">
+          <img src="/src/img/cncf-color.png" alt="We are a Cloud Native Computing Foundation sandbox project">
+          <p>We are a <strong>Cloud Native Computing Foundation</strong> sandbox project.</p>
+        </a>
+      </div>
+    </div>
+    <div class="row">
+      <div class="small-12 medium-12 large-12 columns copyright">
+        Copyright The Porter Authors. All rights reserved. The Linux Foundation has registered trademarks and uses
```

Comment:
should "All rights reserved" be included? I might be confused as how this applies in open source but I thought it's intentionally open source but copyright is separate.

Reply:
Ok, I checked to see how helm does it and https://helm.sh/ and they've got 2 lines 

```
© Helm Authors 2020 | Documentation distributed under CC-BY-4.0

© 2020 The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page.
```

so it makes clear the separation of the documentation license versus the all rights reserved (which I totally don't know what it applies to in this case)

Reply:
Ok found https://github.com/cncf/foundation/blob/f465b352c5e229c6812777fb7821d088d0365d3d/website-guidelines.md 

```CNCF requests that graduated and incubating projects include the sentence “We are a Cloud Native Computing Foundation project.” and the CNCF logo near the bottom of their project homepage. Sandbox-level projects should include the sentence “We are a Cloud Native Computing Foundation sandbox project.” and the CNCF logo. We also appreciate a link to KubeCon + CloudNativeCon as the events approach.
All projects must contain a website footer message referencing trademark guidelines, "Copyright The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page."``` so the All Rights Reserved requirement is associated with the trademark but I'm super confused and not sure what we are saying with it there.

Reply:
I copied nearly verbatim what I was provided as a [good example](https://www.envoyproxy.io/) is all. Following the direction the LF documentation I did not use the copyright symbol or year. If you have questions beyond what I copied from the example they provided me, or about how it's rendered, I'm not the best person to answer since I am just complying with the directions from the LF.

Reply:
Okay I see what you are saying now. Sorry about that! Let me split into two lines like helm so that it's one statement about our copyright and another about the trademark.